### PR TITLE
Trim excel cells content

### PR DESCRIPTION
**Pull request by mmenestret**
## Summary
Trim Excel cells after they are parsed (a trailing space could in a "source" column could terminate the program since it would not find the linked schema sheet)

### CODE REVIEW

File: src/test/scala/com/ebiznext/comet/schema/generator/SchemaGenSpec.scala [hunk 1]
```diff
@@ -19,6 +19,12 @@ class SchemaGenSpec extends TestHelper {
       result.schemas.size shouldBe 2
     }
 
+    it should "trim leading of trailing spaces in cells contents" in {
+      result.schemas.map(_.name) should contain(
+        "SCHEMA1"
+      ) // while it is "SCHEMA1 " in the excel file
```
Comment (mhdkassir) [inline src/test/scala/com/ebiznext/comet/schema/generator/SchemaGenSpec.scala:25]: I don't see a modification on the excel file, the value was already "SCHEMA1 " (with a space at the end ?)

Comment (mmenestret) [reply src/test/scala/com/ebiznext/comet/schema/generator/SchemaGenSpec.scala:25]: I forgot to add the changed excel file, it's fixed !

**Comment by mmenestret** [comment]
Well there are conflicts left, I'll fix them asap

**Comment by mmenestret** [comment]
And its ok now :)

**Pull request merged**